docker/osconsole/config.sh

```diff
@@ -12,7 +12,7 @@ openshift_config_cluster() {
   "openshift": {
     "oauth_metadata_uri": "/master/.well-known/oauth-authorization-server",
     "oauth_client_id": "${HAWTIO_OAUTH_CLIENT_ID:-hawtio-online}",
-    "scope": "user:info user:check-access role:edit:*",
+    "scope": "user:info user:check-access user:list-projects role:edit:*",
```

Comment:
I wonder why this starts to be needed for hawtio-online v2. What changes require it?

Reply:
It probably was always supposed to be there but I took it out during the migration, perhaps. Looking back at [HO-1.0](https://github.com/hawtio/hawtio-online/blob/1.x-redhat-7.x/docker/osconsole/config.sh#L15) was the path to the solution.